FirstTask.py

```diff
@@ -0,0 +1,147 @@
+        if(option == 1):
+            sys("clear")
+            accno=rand(1,100)+1000
+            name=input('Enter Full Name:')
+            passcode=int(input("Enter Passcode:"))
+            accType=input("Enter Account Type:")
+            balance=int(input("Enter Balance for Account:"))
+            obj.append(Bank(name,passcode,accno,balance,accType))
+            print(f"Account Created Successfull!\nYour Account Number is {accno}\n\n")
+            sleep(5)
+            sys("clear")
+
+        elif option ==2 :
+            sys('clear')
+            accno=int(input('Enter Account Number:'))
+            code=int(input('Enter Passcode:'))
+            chk1 = False
+            for i in range(len(obj)):
+                if accno==obj[i].accountNumber() and code==obj[i].Passcode():
+                    chk1 = True
+                    print("Login Successful!\n\n")
+
+                    while(1):
+                        sys('clear')
+                        select=int(input('1) Check Balance \n2) Deposit Bill \n3) Draw Money \n4) Transfer Money \n5) Account Type\n6) Compare Your Account \n0) Exit\nChoose:'))
+                        if select == 1:#checkbalance
+                            print(obj[i].checkBalance())
+                            input()
+                        elif select == 2:#depositbill
+                            dep=int(input("Enter Deposit Amount:"))
+                            obj[i].depositBill(dep)
+                            print("Successfully money deposit!")
+                            input()
+                        elif select == 3:#draw money
+                            amount=int(input('Enter Amount to with draw:'))
+                            if(obj[i].drawMoney(amount)):
+                                print("Money Draw Successful!")
+                            else:
+                                print('Not enough cash!')
+                            input()
+                        elif select == 4:#transfercash
+                            money=int(input('Enter Transfer amount:'))
+                            if( obj[i].transferMoney(money)):
+                                print('Successful Money Transfer!')
+                            else:
+                                print("Not enough cash!")
+                            input()
+                        elif select == 5:
+                            print(obj[i].checkAccountType())
+                            input()
+                        elif select == 6:
+                            if len(obj) > 1:
+                                chk2 = False
+
+                                system('cls')
+                                accNum = int(input("Enter Account Number you want to be compared with: "))
+                                for j in range(len(obj)):
+                                    if accNum == obj[j].accountNumber():
+                                        chk2 = True

```

Comment:
Same here. A better approach here would be to find the other account first and then compare, not do it in a single loop. We can create a static method in the `Account` class to find and return the object from a list of objects using the account number.